fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_clearCapabilityStatement.vail

```diff
@@ -0,0 +1,4 @@
+package com.vantiq.fhir
```

Comment:
What follows here is the contents of the assembly

There will be a pass in a subsequent PR to more fully document things and add a API document and/or READme.  For the present, trying to get basics to work